transports/sql/index.md

```diff
@@ -20,16 +20,17 @@ The SQL Server transport implements a message queuing mechanism on top of [Micro
 
 ## Transport at a glance
 
-|Feature                    |   |
-|:---                       |---
-|Transactions |None, ReceiveOnly, SendsAtomicWithReceive, TransactionScope
-|Pub/Sub                    |Native
-|Timeouts                   |Native
-|Large message bodies       |SqlServer can handle arbitrary message size within available resources, very large messages via data bus
-|Scale-out             |Competing consumer
-|Scripted Deployment        |Sql Scripts
-|Installers                 |Optional
-|Native integration         |[Supported](native-integration.md)
+| Feature                                            |                                                                                                          |
+|:---------------------------------------------------|----------------------------------------------------------------------------------------------------------|
+| Transactions                                       | None, ReceiveOnly, SendsAtomicWithReceive, TransactionScope                                              |
+| Pub/Sub                                            | Native                                                                                                   |
+| Timeouts                                           | Native                                                                                                   |
+| Large message bodies                               | SqlServer can handle arbitrary message size within available resources, very large messages via data bus |
+| Scale-out                                          | Competing consumer                                                                                       |
+| Scripted Deployment                                | Sql Scripts                                                                                              |
+| Installers                                         | Optional                                                                                                 |
+| Native integration                                 | [Supported](native-integration.md)                                                                       |
+| [time-to-be-received](#time-to-be-received) (TTBR) | Storage reclaimed at most 5 minutes after expiration when receiving endpoint is running                  |
```

Comment:
```suggestion
| [time-to-be-received](#time-to-be-received) (TTBR) | Storage reclaimed at most 5 minutes after expiration when receiving endpoint is running                  |
```

Why `5` min? Sql Server transport is purging expired messages on startup, and PostgreSQL is doing that only when messages are received.

Reply:
See reasoning in my other comments. So it seems behavior changed somewhere and documentation was not update to reflect that. I'll update this to ensure both old and new behavior is documented correctly.